### PR TITLE
Extended Variables Support: Fix types in action parameters

### PR DESCRIPTION
**Pull request by Bouh**
Tested and approved:
https://forum.gdevelop.io/t/solved-extended-variables-bug/42390

### CODE REVIEW

File: extensions/reviewed/ExtendedVariables.json [hunk 1]
```diff
@@ -522,10 +468,8 @@
       "sentence": "Change the text of scene variable _PARAM1_ to _PARAM2_",
       "events": [
         {
-          "disabled": false,
-          "folded": false,
           "type": "BuiltinCommonInstructions::JsCode",
-          "inlineCode": "const variable = runtimeScene.getVariables().get(eventsFunctionContext.getArgument(\"SceneVariableName\"));\r\nvariable.setString(eventsFunctionContext.getArgument(\"SceneVariableValue\"));",
+          "inlineCode": "const variable = runtimeScene.getVariables().get(eventsFunctionContext.getArgument(\"SceneVariableName\"));\r\nvariable.setString(eventsFunctionContext.getArgument(\"SceneVariableValue\"));\r\ndebugger;",
```
Comment (D8H) [inline extensions/reviewed/ExtendedVariables.json:472]: What does `debugger;` do?

File: extensions/reviewed/ExtendedVariables.json [hunk 2]
```diff
@@ -1072,7 +981,7 @@
         {
           "codeOnly": false,
           "defaultValue": "yes",
-          "description": "Clear destination first",
+          "description": "Exact copy?",
           "longDescription": "",
           "name": "isExactCopy",
```
Comment (D8H) [inline extensions/reviewed/ExtendedVariables.json:986]: Why this change? What does this parameter do?

File: extensions/reviewed/ExtendedVariables.json [hunk 3]
```diff
@@ -9,7 +9,7 @@
   "name": "ExtendedVariables",
   "previewIconUrl": "https://resources.gdevelop-app.com/assets/Icons/variable-box.svg",
   "shortDescription": "Add conditions, actions and expressions to check for the existence of a variable, copy variables, delete existing ones from memory, and create dynamic variables.",
-  "version": "1.0.0",
+  "version": "1.1.0",
```
Comment (D8H) [inline extensions/reviewed/ExtendedVariables.json:12]: If this is a fix without any new features, it should be a 1.0.1

File: extensions/reviewed/ExtendedVariables.json (hunk 1 above)
Comment (Bouh) [reply extensions/reviewed/ExtendedVariables.json:472]: Damn, good catch!

Comment (Bouh) [inline extensions/reviewed/ExtendedVariables.json:472]:
```suggestion
          "inlineCode": "const variable = runtimeScene.getVariables().get(eventsFunctionContext.getArgument(\"SceneVariableName\"));\r\nvariable.setString(eventsFunctionContext.getArgument(\"SceneVariableValue\"));;",
```

File: extensions/reviewed/ExtendedVariables.json (hunk 2 above)
Comment (Bouh) [reply extensions/reviewed/ExtendedVariables.json:986]: Because it's unclear.
Exact copy mean that copy everything, it does not append or prepend stuff to the existing variable.

File: extensions/reviewed/ExtendedVariables.json (hunk 3 above)
Comment (Bouh) [inline extensions/reviewed/ExtendedVariables.json:12]:
```suggestion
  "version": "1.0.1",
```

File: extensions/reviewed/ExtendedVariables.json (hunk 2 above)
Comment (D8H) [reply extensions/reviewed/ExtendedVariables.json:986]: `Clear destination first` seems to better describe what you've just said.
I'm not against a rephrasing though.
Also, isn't `?` against the good practices?

Comment (Bouh) [reply extensions/reviewed/ExtendedVariables.json:986]: `Clear the variable before copying` ?

**Comment by Bouh** [comment]
Ok to add destination! Let me do that 😁

File: extensions/reviewed/ExtendedVariables.json [hunk 4]
```diff
@@ -1072,7 +981,7 @@
         {
           "codeOnly": false,
           "defaultValue": "yes",
-          "description": "Clear destination first",
+          "description": "Exact copy?",
```
Comment (Bouh) [inline extensions/reviewed/ExtendedVariables.json:984]:
```suggestion
          "description": "Clear the destination variable before copying",
```

File: extensions/reviewed/ExtendedVariables.json [hunk 5]
```diff
@@ -1182,7 +1084,7 @@
...
```
Comment (Bouh) [inline extensions/reviewed/ExtendedVariables.json:1087]:
```suggestion
          "description": "Clear the destination variable before copying",
```

File: extensions/reviewed/ExtendedVariables.json [hunk 6]
```diff
@@ -1410,7 +1300,7 @@
...
```
Comment (Bouh) [inline extensions/reviewed/ExtendedVariables.json:1303]:
```suggestion
          "description": "Clear the destination variable before copying",
```

File: extensions/reviewed/ExtendedVariables.json [hunk 7]
```diff
@@ -1463,7 +1351,7 @@
...
```
Comment (Bouh) [inline extensions/reviewed/ExtendedVariables.json:1354]:
```suggestion
          "description": "Clear the destination variable before copying",
```

File: extensions/reviewed/ExtendedVariables.json (hunk 2 above)
Comment (Bouh) [reply extensions/reviewed/ExtendedVariables.json]: Ok, I changed to 
`Clear the destination variable before copying`

**Review by D8H — APPROVED** [review]
